run.py

```diff
@@ -1,9 +1,15 @@
-from app import app, cli
-from app.models import Category, Language, Resource, db
+from app import app, cli, admin
+from app.models import Category, Language, Resource, db, User, Role
+from flask_security import Security, SQLAlchemyUserDatastore
 from werkzeug.middleware.dispatcher import DispatcherMiddleware
 from prometheus_client import make_wsgi_app
 
 
+admin.run_flask_admin(app)
+
+user_datastore = SQLAlchemyUserDatastore(db, User, Role)
+security = Security(app, user_datastore)
+
```

Comment:
Down on line 26, please add `User` and `Role` to `make_shell_context` so we can reference those models from the flask shell